src/main/java/org/usfirst/frc/team1758/robot/commands/ApproachPeg.java

```diff
@@ -3,78 +3,73 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.usfirst.frc.team1758.robot.RobotMap;
-import org.usfirst.frc.team1758.utilities.TechnoPID;
 
 public class ApproachPeg extends CommandBase {
-	private boolean finished;
-	private TechnoPID pid;
-	private Logger logger;
-	private int counter;
+  private boolean finished;
+  private Logger logger;
+  private int counter;
 
-	public ApproachPeg() {
-		logger = LoggerFactory.getLogger(this.getClass());
-		logger.debug("ApproachPeg command created");
-		requires(vision);
-		requires(driveTrain);
-		requires(sensors);
-	}
+  public ApproachPeg() {
+    logger = LoggerFactory.getLogger(this.getClass());
+    logger.debug("ApproachPeg command created");
+    requires(vision);
+    requires(driveTrain);
+    requires(sensors);
+  }
 
-	protected void initialize() {
-		pid = new TechnoPID(1.6, 0, 0, 5.0);
-		pid.setReference(RobotMap.CAMERA_WIDTH / 2);
-		sensors.resetGyroAngle();
-		finished = false;
-		counter = 0;
-	}
+  protected void initialize() {
+    sensors.resetGyroAngle();
+    finished = false;
+    counter = 0;
+  }
 
-	protected void execute() {
-		logger.trace("Gyro: {} Center: {}", sensors.getUltrasonicValue(), isCentered());
-		if (isDone()) {
-			finished = true;
-			driveTrain.mecanumDriveCartesian(0, 0, 0, 0);
-		} else {
-			iterate();
-		}
+  protected void execute() {
+    logger.trace("Gyro: {} Center: {}", sensors.getUltrasonicValue(), isCentered());
+    if (isDone()) {
+      finished = true;
+      logger.trace("ApproachPeg done");
+      driveTrain.mecanumDriveCartesian(0, 0, 0, 0);
+    } else {
+      iterate();
+    }
 
-	}
+  }
 
-	public void iterate() {
-		double x, y, rotate;
-		x = 0;
-		y = 0;
-		rotate = 0;
-		if (sensors.getUltrasonicValue() > 30) {
-			x = -0.4;
-		}
-		if (!isCentered()) {
-			y = (vision.getCenterX() - RobotMap.CAMERA_WIDTH / 2) / (-3 * (RobotMap.CAMERA_WIDTH / 2));
-		}
-		//logger.trace("Normalized: {}", normalized);
-		logger.trace("Angle: {}", .3 * sensors.getGyroAngle());
-		driveTrain.mecanumDriveCartesian(x, y, rotate, sensors.getGyroAngle());
-	}
+  public void iterate() {
+    double y = 0; 
+    if (sensors.getUltrasonicValue() > 40) {
+      y = 0.3;
+    }
+    double x = 0;
+    if (!isCentered()) {
+      x = (vision.getCenterX() - RobotMap.CAMERA_WIDTH / 2) / (-2 * (RobotMap.CAMERA_WIDTH / 2));
+    }
+    double rotate = 0;
+    logger.trace("Angle: {}", .3 * sensors.getGyroAngle());
+    driveTrain.mecanumDriveCartesian(.75 * x,.75 * y, rotate, 0);
```

Comment:
I really don't like this... but it is working can we multiply the actual values by .75 instead of here?